### DISCUSSION TITLE
API How to get Screenshots and use them in my HTML

### DISCUSSION THREAD

**Discussion by Martin456654** [GoogleChrome/lighthouse#16227] [Q&A]
Hello! I’m currently working with JSON data in my Vue page for my admin panel. I want to utilize screenshots from the JSON structure located at audits > image-alt > details > items, but I'm unsure how to access or generate these screenshots. Any guidance would be greatly appreciated!
![image](https://github.com/user-attachments/assets/bf647399-67ec-46d4-b22b-770a4e3d140e)

**Comment by connorjclark** [comment]
the full page screenshot is at`lhr.fullPageScreenshot`.

some items in the report have a `boundingRect` property - you can use that to clip the `lhr.fullPageScreenshot` and highlight a specific element. You can see how we do it [here](https://github.com/GoogleChrome/lighthouse/blob/main/report/renderer/element-screenshot-renderer.js). probably possible to reuse our code too. good luck.
